users/reverseproxy.rst

```diff
@@ -46,6 +46,21 @@ Nginx
       proxy_pass              http://localhost:8384/;
     }
 
+Caddy
+~~~~~
+
+.. code-block:: nginx
```

Comment:
Nitpicking: Doesn't look like Caddy uses the same config structure as nginx, but is also not known by sphinx itself, so should probably just be dropped.